src/main/kotlin/org/malachite/estella/commons/models/tasks/Task.kt

```diff
@@ -1,15 +1,13 @@
 package org.malachite.estella.commons.models.tasks
 
-import java.sql.Blob
-import java.sql.Clob
 import javax.persistence.*
 
 @Entity
 @Table(name = "tasks")
 data class Task(
         @Id @GeneratedValue(strategy = GenerationType.AUTO) val id: Int?,
-        @Lob val tests: Blob,
-        @Lob val description: Clob,
+        @Lob val tests: Array<Byte>,
```

Comment:
Może i głupie pytanie, ale trochę rozwalasz tym konwencje. Jeśli chcesz to zmienić, to raczej powinieneś to zmieniać wszędzie, a nie tylko w taskach.

Reply:
Nie sądzę, żeby zmiana typu z jakiegoś Sql-owego wrappera na natywny typ było rozwalaniem konwencji. Łatwiej się z tego korzysta w ten sposób, ale nietestowane jeszcze na p o t ę ż n y m deployu, więc nie orałem całego codebasu. Jak się sprawdzi mogę pozmieniać